metron-interface/metron-alerts/e2e/utils/e2e_util.ts

```diff
@@ -46,25 +48,53 @@ export function waitForStalenessOf (_element ) {
 }
 
 export function loadTestData() {
```

Comment:
I think it is important that these test functions are commented and documented.  Not only for maintaining them, but they will possibly be examples for other efforts.

Some of these functions around setting up tests and tearing them down would themselves be useful as utility functions more general than the end to end possibly

Reply:
Agreed